webrtc.html

```diff
@@ -1212,7 +1212,7 @@
             <var>connection</var>.</p>
           </li>
           <li>
-            <p>If <var>newState</var> is <code>"completed"</code>, fire an event
+            <p>If <var>newState</var> is <code>"completed"</code>, <a>fire an ice candidate event</a>
```

Comment:
nit: It sounds a little redundant to say "fire an ice candidate event named `icecandidate`".

This may be unavoidable. But is there a reason we need to specify the event name here, and not in the "fire an ice candidate event" section? The name will always be "icecandidate" (correct?), so it seems like we could remove the "event name" argument entirely, and reference the name "icecandidate" in the "fire an ice candidate event" section instead of here.

Reply:
In most cases it's redundant. But sometimes there are two types/names of events that use the same interface. For example, "addtrack" and "removetrack".

Reply:
Ok, makes sense. If we have to do that for the "track" event, might as well be consistent.